src/org/ensembl/healthcheck/testgroup/ComparaPanIntegrity.java

```diff
@@ -0,0 +1,89 @@
+/*
+ * Copyright [1999-2015] Wellcome Trust Sanger Institute and the EMBL-European Bioinformatics Institute
+ * Copyright [2016-2018] EMBL-European Bioinformatics Institute
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.ensembl.healthcheck.testgroup;
+
+import org.ensembl.healthcheck.GroupOfTests;
+
+/**
+ * Healthchecks for the genomic Compara databases
+ * (and only this kind of database)
+ */
+public class ComparaPanIntegrity extends GroupOfTests {
+
+	public ComparaPanIntegrity() {
+
+		addTest(
```

Comment:
Not a deal-breaker, but since this is almost the same as the ComparaIntegrity group, why not extend that, and then add ForeignKeyPanMasterTables and remove any irrelevant ones?

Reply:
Sure that make sense @james-monkeyshines.